api_calls/reference/clients/clients_read-clients_write-scope-response.json

```diff
@@ -10,33 +10,70 @@
       "id": 1,
       "addresses": [
         {
+          "id": 1,
+          "client_id": 1,
+          "label": "default",
           "address1": "Kutch Stravenue",
           "address2": "Building 1",
           "city": "North Jacey",
-          "country_code": "US",
-          "label": "default",
           "state": "Illinois",
-          "zip": "01363-9579"
+          "zip": "01363-9579",
+          "country_code": "US",
+          "created_at": "2018-12-04T12:33:36Z",
+          "updated_at": "2018-12-04T12:33:36Z",
+          "canceled_at": null,
```

Comment:
not sure if we should expose it since we return only the visible ones, so it's always going to be null, but maybe that's not a big deal

Reply:
well, it's there now... 😅